src/modules/hardwareWallet/components/HardwareAccountManagerModal/HardwareAccountManagerModal.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+import Dialog from '@theme/dialog/dialog';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { useCurrentAccount } from '@account/hooks';
+import { updateHWAccount } from '@hardwareWallet/store/actions';
+import Spinner from '@theme/Spinner';
+import AccountRow from '@account/components/AccountRow';
+import { TertiaryButton } from '@theme/buttons';
+import useHWAccounts from '@hardwareWallet/hooks/useHWAccounts';
+import { selectCurrentHWDevice } from '@hardwareWallet/store/selectors/hwSelectors';
+import Icon from '@theme/Icon';
+import styles from './HardwareAccountManagerModal.css';
+
+// eslint-disable-next-line max-statements
+function HardwareAccountManagerModal() {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [, setCurrentAccount] = useCurrentAccount();
+
+  const [nrOfAccounts, setNrOfAccounts] = useState(0);
+  const { hwAccounts, isLoading } = useHWAccounts(nrOfAccounts);
+
+  const currentHWDevice = useSelector(selectCurrentHWDevice);
+  const { isAppOpen } = currentHWDevice;
+
+  function onSelect(account) {
+    dispatch(updateHWAccount(account));
+    setCurrentAccount(account);
+  }
+
+  function onLoadMore() {
+    setNrOfAccounts((prevState) => prevState + 3);
```

Comment:
define the number as constant which can be later easily modified from one place

Reply:
https://github.com/LiskHQ/lisk-desktop/pull/5276/commits/01cba24e9db045e1ea9796abcc3a4b7ffc46d9f7